src/utilities/oas-operation/oas-operation.ts

```diff
@@ -9,19 +10,26 @@ class OASOperation {
 
   private _exampleGroups: ExampleGroup[];
 
+  private _security: OASSecurity[];
+
   readonly test: string[];
 
   constructor(schema: OperationObject) {
     this.test = [];
     this._schema = schema;
     this.operationId = schema.operationId;
     this._exampleGroups = ExampleGroupFactory.buildFromOperation(this);
+    this._security = OASSecurityFactory.getSecurities(schema.security);
```

Comment:
what do you think about passing the default 'Top' securities into the constructor here and setting those at the operation level if the operation doesn't have any securities set? That would let us remove a bunch of the security logic from the command level.